src/py/flwr/cli/new/templates/app/code/server.xgboost.py.tpl

```diff
@@ -0,0 +1,50 @@
+"""$project_name: A Flower / XGBoost app."""
+
+from typing import Dict
+
+from flwr.common import Context
+from flwr.server import ServerApp, ServerAppComponents, ServerConfig
+from flwr.server.strategy import FedXgbBagging
+
+
+def evaluate_metrics_aggregation(eval_metrics):
+    """Return an aggregated metric (AUC) for evaluation."""
+    total_num = sum([num for num, _ in eval_metrics])
+    auc_aggregated = (
+        sum([metrics["AUC"] * num for num, metrics in eval_metrics]) / total_num
+    )
+    metrics_aggregated = {"AUC": auc_aggregated}
+    return metrics_aggregated
+
+
+def config_func(rnd: int) -> Dict[str, str]:
+    """Return a configuration with global epochs."""
+    config = {
+        "global_round": str(rnd),
+    }
+    return config
+
+
+def server_fn(context: Context):
+    # Read from config
+    num_rounds = context.run_config["num-server-rounds"]
+    fraction_fit = context.run_config["fraction-fit"]
+    fraction_evaluate = context.run_config["fraction-evaluate"]
+
```

Comment:
is it possible to initialize the global model here? This is something we'd want to do for all templates.

Reply:
The `XGBoost` object is different from NN training from my understanding. Without calling the `train()` function, the `XGBoost` object doesn't contain any model parameters (only meta info). I think we don't want to train it on the server side before FL?

Reply:
i see. maybe then it's enough to do what `get_parameters()` does (i.e. send and empty array) on the `ServerApp`?:

```python
# we can do this on the serverapp and pass it to the strategy.
# then remove the `get_parameters()`
parameters=Parameters(tensor_type="", tensors=[])
```

Reply:
We decided to remove `get_parameters()` defined in client and keep the server as it is